.travis.yml

```diff
@@ -1,18 +1,18 @@
 # Config file for automatic testing at travis-ci.org
-
+dist: xenial
 sudo: false
 language: python
 
 matrix:
     include:
       - python: 2.7
         env: TOXENV=py27
-      - python: 3.4
-        env: TOXENV=py34
       - python: 3.5
         env: TOXENV=py35
       - python: 3.6
         env: TOXENV=py36
+      - python: 3.7
+        env: TOXENV=py37
       - python: pypy
         env: TOXENV=pypy
       - python: 3.5
```

Comment:
flake8 should be run on Python 3.7 instead of 3.5 to catch the new syntax errors associated with __async__ and __await__ and new deprecation warnings for Python 3.8

Reply:
You are right but I would merge this in current form and pospone any other fine tunning for later. The totally broken CI is far more damaging that these small details.

I am saying this because the change because there is no ending regarding number of fixes/improvements.

Another note: I would personally disable CodeCov as I find it more of a source of noise than somethign useful.